package.json

```diff
@@ -149,6 +149,11 @@
                 "type": "string",
                 "description": "The source root on this machine that is the equivalent to the serverSourceRoot on the server."
               },
+              "pathMappings": {
+                "type": "object",
+                "default": {},
+                "description": "A mapping of multiple localSourceRoots on this machine to remote serverSourceRoots on the server."
```

Comment:
I'm not sure what you mean by "the old settings should be deprecated". I'm assuming I should just add "Deprecated: " to the beginning of the description?